resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -1,269 +1,411 @@
+                            @endphp
+                            <div class="col-sm-4">
+                              <label for="LoadFare{{ $fare->id }}">{{ $fare->name }} Load [
+                                Max: {{ number_format($fare->capacity) }} ]</label>
+                              <input id="LoadFare{{ $fare->id }}" type="text" class="form-control"
+                                     value="{{ number_format($randomloadperfare) }} @if($randomloadperfare > '900') {{ setting('units.weight') }} @endif"
+                                     disabled/>
+                            </div>
+                          @endif
+                        @endforeach
+                        @php
+                          $loadcollection = collect($loadarray) ;
+                          $totalgenload = $loadcollection->sum('SeatLoad') ;
+                        @endphp
+                      </div>
+
+                      @php $pxweight = '208' ; @endphp
+                      @if($totalgenload > 0 && $totalgenload < 900)
+                        <input type="hidden" name="acdata" value="{'paxwgt':{{ $pax_weight }}">
+                        <br>
+                        <div class="row">
+                          <div class="col-sm-4">
+                            @if(setting('units.weight') === 'kg')
+                              @php $estimatedpayload = number_format(round(($pax_weight * $totalgenload) / 2.2)) ; @endphp
+                            @else
+                              @php $estimatedpayload = number_format(round($pax_weight * $totalgenload)) ; @endphp
+                            @endif
+                            <label for="EstimatedLoad">Estimated Payload For {{ $totalgenload }} Pax</label>
+                            <input id="EstimatedLoad" type="text" class="form-control"
+                                   value="{{ $estimatedpayload }} {{ setting('units.weight') }}" disabled/>
+                          </div>
+                        </div>
+                        <input type="hidden" id="pax" name="pax" class="form-control" value="{{ $totalgenload }}"/>
+                      @elseif($totalgenload > 900)
+                        <input type='hidden' id="pax" name='pax' value='0' maxlength='3'>
+                        <input type='hidden' id="cargo" name='cargo' value="{{ $totalgenload }}" maxlength='7'>
+                      @endif
+                    @endif
+                  @endforeach
+                  <br>
+                  <div class="row">
+                    @php
+                      $flightype = 'SimBrief Standard';
+                      if($flight->flight_type == 'C') { $flightype = 'Charter All Adult Pax' ;}
+                      if($flight->flight_type == 'J' || $flight->flight_type == 'G') { $flightype = 'Schedule All Adult Pax' ;}
+                      if($flight->flight_type == 'F' || $flight->flight_type == 'A' || $flight->flight_type == 'H') { $flightype = 'Only Cargo' ;}
```

Comment:
This should all use the language strings - so below should read `{{ FlightType::label($flight->flight_type) }}`

Reply:
Why ? I am not using the flight type here 

This is where I build an info text according to the flight type , I agree on using language files but then we need to add 'Schedule All Adult Pax' this to the language pack.

Or this can be completely removed , it is just an info text displayed to give the end user info about which weights will be used in his simbrief flight plan

Reply:
Is "Schedule All Adult Pax" industry language? Just asking because I'm not familiar with it. If you want it to be here then I think it makes sense to then add it to the language files. There is language there now... ex, `lang/en/flights.php`, "Passenger - Scheduled" - could that be used? "Passenger - Scheduled All Adult Pax", for example?

Reply:
Well, the actually industry specific name is "All flights except holiday charters" but it is like the VMC/IMC thing, on daily basis people use "Scheduled" or "Charter" mostly , and it can be varied as "All Adult" or "Male/Female/Children" ... 

![mb](https://user-images.githubusercontent.com/74361521/102833485-4bf67b00-4402-11eb-9d7d-1bc792a06ffb.png)

Reply:
Yeah , it is just an info text after all , may be removed I can handle it on my side :) No need to go this industry specific

Reply:
Removed the info text completely , will handle it on my site personally